<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardGameMain</class>
 <widget class="QMainWindow" name="CardGameMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>741</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CardGameMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonStartGame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxUserHand">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>831</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Your Hand</string>
    </property>
    <widget class="QFrame" name="frameUserCard1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonUserCard1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelUserCard1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelUserCard1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelUserCard2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelUserCard2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameUserCard2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonUserCard2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelUserCard3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelUserCard3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameUserCard3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonUserCard3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelUserCard4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>labelUserCard4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameUserCard4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonUserCard4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelUserCard5">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>labelUserCard5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameUserCard5">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonUserCard5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelUserCategory">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>User's hand Category</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelCardRestText">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rest:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSwap">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Swap</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxRobotHand">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>831</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Robot's hand</string>
    </property>
    <widget class="QFrame" name="frameRobotCard1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonRobotCard1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelRobotCard1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelRobotCard1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelRobotCard2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelRobotCard2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameRobotCard2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonRobotCard2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelRobotCard3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelRobotCard3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameRobotCard3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonRobotCard3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelRobotCard4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelRobotCard4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameRobotCard4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonRobotCard4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelRobotCard5">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>labelRobotCard5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frameRobotCard5">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonRobotCard5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>131</width>
        <height>165</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelRobotCategory">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Robot's hand Category</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelRoundText">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Round: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserScoreText">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRobotScoreText">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Robot: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRound">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserScore">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRobotScore">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCardRest">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>52</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNextRound">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Next round</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWinner">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Winner!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCardDeck">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>300</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGameOver">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Game Over</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConfirm">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReset_Game">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionReset_Game"/>
    <addaction name="actionSet_Player_s_name"/>
    <addaction name="actionAbout_me"/>
   </widget>
   <addaction name="menuReset_Game"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionSet_Player_s_name">
   <property name="text">
    <string>Set Player's Name</string>
   </property>
  </action>
  <action name="actionAbout_me">
   <property name="text">
    <string>About me</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
